### PR TITLE
Added Remove Button Near To Edit Button In Shortcut Items

### PR DESCRIPTION
**Pull request by AmbrishRamachandiran**
## Hey, I just made a Pull Request!

When a user uses the Add Shortcut button to create several shortcuts, a list will be created; however, under the current functionality, a user must first click on Edit before they may delete a shortcut.
**So placed the remove button and icon close to the edit button so that users can quickly remove the shortcut.**

     
image

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by AmbrishRamachandiran** [comment]
the commit is for https://github.com/backstage/backstage/issues/18706

File: .changeset/spotty-boats-unite.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-shortcuts': major
```
Comment (benjdlambert) [inline .changeset/spotty-boats-unite.md:2]: This should be `patch` change not `minor`

File: plugins/shortcuts/src/ShortcutItem.tsx [hunk 2]
```diff
@@ -94,6 +117,14 @@ export const ShortcutItem = ({ shortcut, api, allowExternalLinks }: Props) => {
           >
             <EditIcon className={classes.icon} />
           </IconButton>
+          <IconButton
```
Comment (benjdlambert) [inline plugins/shortcuts/src/ShortcutItem.tsx:120]: I think that like @ahhhndre said, we should make this opt in. Theres not a lot of room there in the first place.

File: plugins/shortcuts/src/ShortcutItem.tsx [hunk 3]
```diff
@@ -19,20 +19,26 @@ import { makeStyles } from '@material-ui/core/styles';
...
     '&:hover #edit': {
       visibility: 'visible',
     },
+    '&:hover #remove': {
+      visibility: 'visible',
+    },
   },
   button: {
     visibility: 'hidden',
+    padding: 3,
```
Comment (benjdlambert) [inline plugins/shortcuts/src/ShortcutItem.tsx:41]: Think that we should use `theme.spacing` here instead.

**Review by benjdlambert — CHANGES_REQUESTED** [review]

**Review by AmbrishRamachandiran — COMMENTED** [review]
changes added

**Review by AmbrishRamachandiran — COMMENTED** [review]
Updated files as per review @benjdlambert

File: plugins/shortcuts/src/ShortcutItem.tsx (hunk 2 above)
Comment (AmbrishRamachandiran) [reply plugins/shortcuts/src/ShortcutItem.tsx:120]: yes !! made it opt in working fine please review

**Review by AmbrishRamachandiran — COMMENTED** [review]
@benjdlambert this latest commit have all the changes as per your review comments , please review it and let me know if any other changes are required thanks in advance.

**Comment by freben** [comment]
Might I suggest to add a three-dots menu instead that opens a small popover with the delete and edit actions? And that menu button should ideally only be visible on hover. This takes up both less physical space and less mental overhead.

**Comment by AmbrishRamachandiran** [comment]
@freben adding three dots & showing it will take more effort , will work on this later for now closing the PR

**Pull request closed without merging**